src/theme-provider.js

```diff
@@ -29,6 +30,7 @@ class ThemeProvider extends Component {
 
   setOuterTheme = theme => {
     this.outerTheme = theme
+    this.broadcast.setState(this.getTheme())
```

Comment:
Moved this from `componentWillMount`. I feel like without this here, anytime the emitter broadcasted an update, this component wouldn't re-render. But I could also be totally confused 😅